Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN dotnet restore Libplanet.Explorer.Executable
 
 # Copy everything else and build
 COPY . ./
-RUN dotnet publish -c Release -r linux-x64 -o out --self-contained
+RUN dotnet publish Libplanet.Explorer -c Release -r linux-x64 -o out --self-contained
+RUN dotnet publish Libplanet.Explorer.Executable -c Release -r linux-x64 -o out --self-contained
```

Comment:
Without this isn't it built successfully?

Reply:
Since solution file contains `Libplanet.Explorer.UnitTests`, just publish all projects will cause an error.

Reply:
Doesn't adding the test project into *.dockerignore* list solve that?

Reply:
I guess leaving publish target argument empty, it will automatically runs .sln file, not searching all projects. Since `Libplanet.Explorer.sln` contains `Libplanet.Explorer.UnitTests`, it cannot find corresponding files so cause error.

Reply:
Fair enough.  So would it work even if we roll back *.dockerignore*?

Reply:
Yes. I added them to *.dockerignore* just to minimize image size. :)